CareKitStore/CareKitStore/HealthKit/OCKHealthKitPassthroughStore+EventUtilities.swift

```diff
@@ -562,7 +562,10 @@ extension OCKHealthKitPassthroughStore {
 
     func makeTaskQuery(from outcomeQuery: OCKOutcomeQuery) -> OCKTaskQuery {
 
-        let dateInterval = Calendar.current.dateInterval(of: .day, for: Date())!
+        // Search over the interval provided by OCKOutcomeQuery if present
+        // or else constrain sample query over the current day.
+        let dateInterval = outcomeQuery.dateInterval ??
+            Calendar.current.dateInterval(of: .day, for: Date())!
```

Comment:
Similar to https://github.com/carekit-apple/CareKit/blob/b16d15c543cabfb84954a5a3d7df3aea7c749bcb/CareKitStore/CareKitStore/Protocols/Events/OCKReadOnlyEventStore%2BVersioningUtilities.swift#L225-L232